tests/ui/dropck/const_drop_is_valid.rs

```diff
@@ -1,8 +1,11 @@
+//@ check-pass
+
+#![feature(const_trait_impl)]
+
 struct A();
 
-impl const Drop for A {}
-//~^ ERROR: const trait impls are experimental
-//~| const `impl` for trait `Drop` which is not marked with `#[const_trait]`
-//~| not all trait items implemented, missing: `drop`
+impl const Drop for A {
+    fn drop(&mut self) {}
+}
```

Comment:
that feels like an odd test, why does it exist?

Reply:
sure